src/oss-reproducible/Strategies/PackageMatchesSourceStrategy.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+using NLog;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.CST.OpenSource.Reproducibility
+{
+    /// <summary>
+    /// This strategy checks to see if the package content exactly matches the source code
+    /// repository. It automatically excludes the .git directory.
```

Comment:
Where is the `.git` directory excluded? its not explicitly excluded by options you set or by the DirectoryDifference method as far as I can tell.

Reply:
Good catch, thanks!